dockers/broad/imputation/build_push_docker.sh

```diff
@@ -0,0 +1,119 @@
+set -e
```

Comment:
gave this a look over, since there are multiple dockerfiles I have no problem putting a ticket on the backlog to refactor later. Would probably be faster just to do it that way so we can move this on without worrying about semantics for now